src/helper-methods.js

```diff
@@ -0,0 +1,67 @@
+import merge from "lodash/object/merge";
+import { Chart, Style } from "victory-util";
+import { Transform } from "react-art";
+
+
+module.exports = {
+  getStyles(props, defaultStyles) {
+    const style = props.style ? merge({}, defaultStyles, props.style) : defaultStyles;
+    return Style.removeInvisible(Chart.evaluateStyle(style));
+  },
+
+  getContentLength(props) {
+    if (props.children) {
```

Comment:
^ in general, I think we should strive to have more of our helpers/util functions *not* be passed an entire props object, and instead just be passed the individual parts that are needed. When a props object is passed, it makes the function more brittle if we decide to change the props structure later, and also ties it directly to the structure of components rather than being generally useful functions. Doing this more may have to involve disabling the eslint rule which limits the number of arguments you can pass to a function, which IMHO is not a good rule anyway.